test/config/test_configtypes.py

```diff
@@ -70,72 +67,77 @@ def __repr__(self):
                               password=self.password())
 
 
-class ValidValuesTest(unittest.TestCase):
+@pytest.fixture
+def os_path(mocker):
+    """Fixture that mocks and returns os.path from the configtypes module"""
+    return mocker.patch('qutebrowser.config.configtypes.os.path', autospec=True)
 
+
+class TestValidValues:
     """Test ValidValues."""
 
     def test_contains_without_desc(self):
         """Test __contains__ without a description."""
         vv = configtypes.ValidValues('foo', 'bar')
-        self.assertIn('foo', vv)
-        self.assertNotIn("baz", vv)
+        assert 'foo' in vv
+        assert "baz" not in vv
 
     def test_contains_with_desc(self):
         """Test __contains__ with a description."""
         vv = configtypes.ValidValues(('foo', "foo desc"), ('bar', "bar desc"))
-        self.assertIn('foo', vv)
-        self.assertIn('bar', vv)
-        self.assertNotIn("baz", vv)
+        assert 'foo' in vv
+        assert 'bar' in vv
+        assert "baz" not in vv
 
     def test_contains_mixed_desc(self):
         """Test __contains__ with mixed description."""
         vv = configtypes.ValidValues(('foo', "foo desc"), 'bar')
-        self.assertIn('foo', vv)
-        self.assertIn('bar', vv)
-        self.assertNotIn("baz", vv)
+        assert 'foo' in vv
+        assert 'bar' in vv
+        assert "baz" not in vv
 
     def test_iter_without_desc(self):
         """Test __iter__ without a description."""
         vv = configtypes.ValidValues('foo', 'bar')
-        self.assertEqual(list(vv), ['foo', 'bar'])
+        assert list(vv), ['foo' == 'bar']
```

Comment:
I think this should be `assert list(vv) == ['foo', 'bar']` - if I parse this right, you're basically asserting `(list(vv), [False])` now, which is always true :stuck_out_tongue: